src/commons/stores/cluster-store.js

```diff
@@ -306,6 +306,10 @@
                 status = "Pending";
             } else {
                 status = profileStatus[cluster.volume_profiling_state];
+
+                if(typeof status === "undefined") {
+                    status = "Unknown";
```

Comment:
Status coming in lower case so Unknown should be lower case too

Reply:
@cloudbehl its used for showing purpose in UI.